docroot/themes/custom/uids_base/templates/views/node--view--articles-block--block-articles.html.twig

```diff
@@ -0,0 +1,67 @@
+{%
+  set classes = [
+  'node',
+  'node--type-' ~ node.bundle|clean_class,
+  node.isPromoted() ? 'node--promoted',
+  node.isSticky() ? 'node--sticky',
+  not node.isPublished() ? 'node--unpublished',
+  view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+  'media',
+  'card',
+  'card--enclosed',
```

Comment:
@pyrello can we add `card--horizontal` here by default and remove it if the Article Listing Settings is set to `grid` or `masonry`?

Reply:
I think we would need to do that in preprocess. I just added those 'card' classes there to get it done, but I'm not sure its the right approach.